.github/workflows/integration-tests.yml

```diff
@@ -18,12 +18,18 @@ jobs:
         with:
           python-version: "3.8"
 
-      - name: install test requirements
+      - name: Install package dependencies
         run: |
+          sudo apt-get update
+          sudo apt-get --assume-yes --no-install-recommends install libsystemd0 libsystemd-dev pkg-config
```

Comment:
Why install these packages or make any changes to github workflows since you did not add any actual tests for the plugin?

Reply:
There is no new integration added for this plugin. Try amend your PR without these installations and see whether the PR passes the build.